src/Pages/Profile/MyAccount.tsx

```diff
@@ -21,17 +23,37 @@ const MyAccount: React.FC = () => {
 
   useEffect(() => {
     if (user) {
-      setName(user.displayName);
+      if (user.displayName) {
```

Comment:
If I see this correctly, you're dealing with the names stored in the previous format (before the first/last name split) by splitting them on the fly.

Would it make sense to "bake" this process into our database? Essentially going through all names we currently have, split them and store them correctly? Tagging @hugo2410 as well, since it might relate to his territory ^^

Reply:
I believe in the previous set up, the user should have written FirstName followed by the last name, so ideally the outcome should be similar. I tried to check on the DB but i can't see it directly...

Reply:
Sorry for the super delayed response - I hope you had an awesome Christmas! :PP

Totally; I just wonder whether we want to split those names from the previous setup (stored as `displayName`) and run a process over our database, which splits those names in the same way, but then stores the result as `firstName`, `lastName` into the DB, so all users are stored the same way and we don't have to have an if/else when fetching the names.

That way we don't have to worry about it again, for example if we want to use the name in other areas.